CONTRIBUTING.md

```diff
@@ -145,8 +145,10 @@ Cluster API maintains the most recent release/releases for all supported API and
 | **v1alpha4** | EOL since 2022-04-06 |
 | **v1alpha3** | EOL since 2022-02-23 |
 
-- For the latest API version we support the two most recent minor releases; older minor releases are immediately unsupported when a new major/minor release is available. 
+- For the latest API version we support the two most recent minor releases; older minor releases are immediately unsupported when a new major/minor release is available.
 - For older API versions we only support the most recent minor release until the API version reaches EOL.
+- We will maintain test coverage for all supported minor releases and for one additional release for the latest API version in case we have to do an emergency patch release.
```

Comment:
```suggestion
- We will maintain test coverage for all supported minor releases and for one additional release for the current stable API version in case we have to do an emergency patch release.
```